components/script/dom/htmlinputelement.rs

```diff
@@ -1187,6 +1187,13 @@ impl HTMLInputElementMethods for HTMLInputElement {
         self.filelist.get().as_ref().map(|fl| fl.clone())
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-input-files
+    fn SetFiles(&self, files: Option<&FileList>) {
```

Comment:
Let's start using proper rustdoc for these functions:

```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-input-files>
    fn SetFiles(&self, files: Option<&FileList>) {
```